### PR TITLE
[FAQ] explain 2-of-3 merging rule

### PR DESCRIPTION
**Pull request by MaxHillebrand**
An update to fix the explanation of who can merge what.

All of @nopara73 @lontivero and @molnard have admin privileges and can thus merge code all by them selves. However the agreed upon convention is that 2-of-3 agree.

Can you please confirm that this is correct?

### CODE REVIEW

**Comment by nopara73** [comment]
Yes, that's correct. The convention is ensured by GitHub, so it's a bit more than a convention, it's just we're right now next to each other and we're a bit more free about this rule, since if someone says ok in person, then it's a waste of time to send him back to his computer and make him approve formally:)

File: docs/FAQ/FAQ-Contribution.md [hunk 1]
```diff
@@ -44,9 +44,11 @@ Wasabi is cutting-edge, high security software, and there can never be enough ey
...
-- [Lucas Ontivero](https://github.com/lontivero) [lead engineer of [zkSnacks Ltd](https://zksnacks.com/)] is [co-maintainer](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [GitHub repository](https://github.com/zksnacks/walletwasabi), he can merge a pull request, but only when both Ádám and Dávid have reviewed and acknowledged the changes.
-- [Dávid Molnár](https://github.com/molnard) is [co-maintainer](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [GitHub repository](https://github.com/zksnacks/walletwasabi), he can merge a pull request, but only when both Ádám and Lucas have reviewed and acknowledged the changes.
+Per default, they [require at least one review](https://help.github.com/en/github/administering-a-repository/about-required-reviews-for-pull-requests) by one of the other maintainers before a pull request can be merged.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Contribution.md]:
```suggestion
Per default, they [require at least one approving review](https://help.github.com/en/github/administering-a-repository/about-required-reviews-for-pull-requests) by one of the other maintainers before a pull request can be merged.
```

File: docs/FAQ/FAQ-Contribution.md [hunk 2]
```diff
@@ -44,9 +44,11 @@ Wasabi is cutting-edge, high security software, and there can never be enough ey
...
+However, since all three are administrators, they can still force the merge without the approval of others, but this will raise alarm bells.
+- [Ádám Ficsor](https://github.com/nopara73) [co-founder and former CTO of [zkSnacks Ltd](https://zksnacks.com/), co-author of the [zero link Bitcoin fungibility framework](https://github.com/nopara73/ZeroLink)] is the [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [GitHub repository](https://github.com/zksnacks/walletwasabi).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Contribution.md]:
```suggestion
- [Ádám Ficsor](https://github.com/nopara73) [co-founder and former CTO of [zkSNACKs Ltd](https://zksnacks.com/), co-author of the [zero link Bitcoin fungibility framework](https://github.com/nopara73/ZeroLink)] is the [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [Wasabi Wallet repository](https://github.com/zksnacks/walletwasabi).
```

File: docs/FAQ/FAQ-Contribution.md [hunk 3]
```diff
@@ -44,9 +44,11 @@ Wasabi is cutting-edge, high security software, and there can never be enough ey
...
+- [Lucas Ontivero](https://github.com/lontivero) [lead engineer of [zkSnacks Ltd](https://zksnacks.com/)] is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [GitHub repository](https://github.com/zksnacks/walletwasabi).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Contribution.md]:
```suggestion
- [Lucas Ontivero](https://github.com/lontivero) [lead engineer of [zkSNACKs Ltd](https://zksnacks.com/)] is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [Wasabi Wallet repository](https://github.com/zksnacks/walletwasabi).
```

File: docs/FAQ/FAQ-Contribution.md [hunk 4]
```diff
@@ -44,9 +44,11 @@ Wasabi is cutting-edge, high security software, and there can never be enough ey
...
+- [Dávid Molnár](https://github.com/molnard) is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) and administrator of the [GitHub repository](https://github.com/zksnacks/walletwasabi).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Contribution.md]:
```suggestion
- [Dávid Molnár](https://github.com/molnard) [CTO of [zkSNACKs Ltd](https://zksnacks.com/)] is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [Wasabi Wallet repository](https://github.com/zksnacks/walletwasabi).
```

**Comment by MaxHillebrand** [comment]
@nopara73 can you please ACK the re-written changes? 
Then it is ready for merge.

**Pull request merged**